core/src/main/resources/defaults/clustertemplates/7.2.7/gcp/flow-management.json

```diff
@@ -8,6 +8,9 @@
     "cluster": {
       "blueprintName": "7.2.7 - Flow Management Heavy Duty with Apache NiFi, Apache NiFi Registry"
     },
+    "externalDatabase": {
+      "availabilityType": "HA"
```

Comment:
I wonder if HA databases are also supported in GCP yet. If not, this may be ignored silenty (and provision a non-HA DB instead), or it may also cause data hub provisioning abort if the HA availability is expressly forbidden. 😃

Reply:
@lajosrodek  i asked Richard about that and he said that Cloudbreak will support external db for GCP in 2.35 Cloudbreak version. According to "Cloudbreak Release Schedule" (https://cloudera.atlassian.net/wiki/spaces/ENG/pages/610469693/Cloudbreak+Release+Schedule), this functionality should be available on prod in the middle of December, on mow-dev it should be available on the beginning of December. So looks like its actual for 7.2.7.

Reply:
👍